CONTRIBUTING.md

```diff
@@ -5,11 +5,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
 **Get started:**
 - Choose an issue from the [TO-DO](https://github.com/Techtonica/curriculum/projects/2) column that seems right for you & message Techtonica staff. If you know someone connected to Techtonica, they may be able to add you to our slack channel. If you are here for GSSoC19, message Alina L. (M) and say you'd like to pick up the issue.
 - In the issue, you should find a link for a lesson outline and for corresponding slides.
-- If it is in the ["TO-DO"](https://github.com/Techtonica/curriculum/projects/2) column, that means it is available, even if there's an "assignee".
-- To **create an outline**, fork the repo or commit directly onto the master branch.  [Here is a topic outline template that explains what we need for each section](https://github.com/Techtonica/curriculum/blob/master/_templates/topic-outline.md).
+- If it is in the ["TO-DO"](https://github.com/Techtonica/curriculum/blob/master/projects/2) column, that means it is available, even if there's an "assignee".
```

Comment:
Please undo this line. You broke this link.

Reply:
which link i broke ..please mention link?

Reply:
I think he's talking about `https://github.com/Techtonica/curriculum/blob/master/projects/2`

Reply:
yes i got it .. thank you.I changed, Now it's working.So i only corrected CONTRIBUTING.md .ready to send pull request

Reply:
I changed,commited, and again pushed but unable to see any PR request?